plugins/modules/sonic_prefix_lists.py

```diff
@@ -0,0 +1,423 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright 2019 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#############################################
+#                WARNING                    #
+#############################################
+#
+# This file is auto generated by the resource
+#   module builder playbook.
+#
+# Do not edit this file manually.
+#
+# Changes to this file will be over written
+#   by the resource module builder.
+#
+# Changes should be made in the model used to
+#   generate this file or in the resource module
+#   builder template.
+#
+#############################################
+
+"""
+The module file for sonic_prefix_lists
+"""
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = """
+---
+module: sonic_prefix_lists
+version_added: "1.2.0"
+author: Kerry Meyer (@kerry-meyer)
+short_description: prefix list configuration handling for SONiC
+description:
+  - This module provides configuration management for prefix list parameters on devices running SONiC.
+options:
+  config:
+    description:
+      - Specifies a list of prefix set configuration dictionaries
+    type: list
+    elements: dict
+    suboptions:
+      name:
+        description:
+          - name of a prefix set (a list of prefix entries)
```

Comment:
Please capitalize the description for consistency.

Reply:
Done.